tests/mbed/host/host.c

```diff
@@ -37,20 +37,14 @@ char* find_data_file(char* str, size_t size)
 void datafileloc(char* data_file_name, char* path)
 {
     char* tail = "3rdparty/mbedtls/mbedtls/tests/suites/";
-    char* seperator;
+    char* separator = getenv("PROJECT_SOURCE_DIR");
```

Comment:
I am not sure why we have to use an environment variable here (which would break pretty easily if the tests are not run with ctest, I think). Why not just add a #define?